tests/fib/test_fib.py

```diff
@@ -102,31 +115,23 @@ def get_router_interface_ports(config_facts, testbed):
 
     return router_interface_ports
 
-@pytest.mark.parametrize("ipv4, ipv6, mtu", [pytest.param(True, True, 1514)])
-def test_fib(testbed, duthost, ptfhost, ipv4, ipv6, mtu):
 
-    config_facts = duthost.config_facts(host=duthost.hostname, source="running")['ansible_facts']
-
-    t = datetime.now().strftime('%Y-%m-%d-%H:%M:%S')
-
-    ofpname = "/tmp/fib/{}/tmp/fib_info.{}.txt".format(duthost.hostname, t)
-
-    build_fib(duthost, config_facts, ofpname, t)
+@pytest.mark.parametrize("ipv4, ipv6, mtu", [pytest.param(True, True, 1514)])
+def test_basic_fib(testbed, duthost, ptfhost, ipv4, ipv6, mtu, config_facts, build_fib):
 
-    ptfhost.copy(src=ofpname, dest="/root/fib_info.txt")
-    logging.info("run ptf test")
+    timestamp = datetime.now().strftime('%Y-%m-%d-%H:%M:%S')
 
     # do not test load balancing for vs platform as kernel 4.9
     # can only do load balance base on L3
-    meta = config_facts.get('DEVICE_METADATA')
-    if meta['localhost']['platform'] == 'x86_64-kvm_x86_64-r0':
+    if duthost.facts['platform'] == 'x86_64-kvm_x86_64-r0':
```

Comment:
```suggestion
    if duthost.facts['asic'] == 'vs':
```
This seems a little more flexible. Thoughts?

Reply:
Sure, updated.